src/application/response.ts

```diff
@@ -0,0 +1,32 @@
+export namespace PilgrimResponse {
```

Comment:
An experimental module at the moment. Would like to have a response type that is global and is flavoured by the event sources where possible.